wandb/__init__.py

```diff
@@ -98,6 +104,11 @@ def ensure_configured():
     global api
     api = InternalApi()
 
+def set_trace():
+    import pdb  # TODO: support other debuggers
+    #  frame = sys._getframe().f_back
+    pdb.set_trace() # TODO: pass the parent stack...
```

Comment:
I should probably rip this out, but I think there are some cool things we could do here.  pytest exposes `pytest.set_trace` that stops mirroring io and adds some nice stuff.  Someday we'll do some cool stuff here.

Reply:
this is so that users can call `wandb.set_trace` in their code while debugging?